### ISSUE TITLE
ADD: Alert box or Kind of a warning pop-up for error

### ISSUE THREAD

**Issue by Sakhi29** [Aryainguz/Euphoria-Check-PERMA-Meter-Express#9]
On submitting the blank form without selecting any answer to the question, we are being redirected to a page which looks something like this 👇...... So, rather than rendering a new page for error we can just show pop-up message to the user to fill out and answer all the questions or else it won't be submitted.
![image](https://github.com/Aryainguz/Euphoria-Check-PERMA-Meter-Express/assets/106910881/4293965d-3bbd-45f6-a2a8-2a6a149c0119)

I would like to work upon this! Waiting for issuance! Happy Contributing :)

**Comment by Aryainguz** [comment]
Yes @Sakhi29 this sounds great, go ahead and feel free to reach out for any help

**Comment by Anubhab-03** [comment]
I can contribute in implementing the pop up messages section.Kindly assign this to me under SWOC'24

**Comment by kamalsritha** [comment]
can you please assign me this issue under SWOC.

**Comment by Aryainguz** [comment]
Sure go ahead @kamalsritha

**Comment by kamalsritha** [comment]
@Aryainguz I intended to create a separate pull request for this issue, but I encountered some difficulties. My apologies for any inconvenience caused. To keep things moving, I've added a new commit to the existing pull request. I hope you'll be able to review it soon.
